File: src/main/java/com/bvanseg/gigeresque/common/item/group/ItemGroups.java
package com.bvanseg.gigeresque.common.item.group;

import com.bvanseg.gigeresque.common.Gigeresque;
import com.bvanseg.gigeresque.common.block.Blocks;

import net.fabricmc.fabric.api.client.itemgroup.FabricItemGroupBuilder;
import net.minecraft.item.ItemGroup;
import net.minecraft.item.ItemStack;
import net.minecraft.util.Identifier;

public class ItemGroups {
	private ItemGroups() {
	}

	public static final ItemGroup GENERAL = FabricItemGroupBuilder.build(new Identifier(Gigeresque.MOD_ID, "general"),
			() -> new ItemStack(Blocks.NEST_RESIN_WEB));
}

File: src/main/kotlin/com/bvanseg/gigeresque/common/item/group/ItemGroups.kt
package com.bvanseg.gigeresque.common.item.group

import com.bvanseg.gigeresque.common.Gigeresque
import net.fabricmc.fabric.api.client.itemgroup.FabricItemGroupBuilder
import net.minecraft.item.ItemStack
import net.minecraft.item.Items
import net.minecraft.util.Identifier

/**
 * @author Boston Vanseghi
 */
object ItemGroups {
    val GENERAL = FabricItemGroupBuilder.build(
        Identifier(Gigeresque.MOD_ID, "general")
    ) { ItemStack(Items.BOWL) }
}